labelbox/schema/batch.py

```diff
@@ -92,23 +94,27 @@ def export_data_rows(self, timeout_seconds=120) -> Generator:
             LabelboxError: if the export fails or is unable to download within the specified time.
         """
         id_param = "batchId"
-        query_str = """mutation GetBatchDataRowsExportUrlPyApi($%s: ID!)
-            {exportBatchDataRows(data:{batchId: $%s }) {downloadUrl createdAt status}}
-        """ % (id_param, id_param)
+        metadata_param = "includeMetadataInput"
+        query_str = """mutation GetBatchDataRowsExportUrlPyApi($%s: ID!, $%s: Boolean!)
+            {exportBatchDataRows(data:{batchId: $%s , includeMetadataInput: $%s}) {downloadUrl createdAt status}}
+        """ % (id_param, metadata_param, id_param, metadata_param)
         sleep_time = 2
         while True:
-            res = self.client.execute(query_str, {id_param: self.uid})
+            res = self.client.execute(query_str, {
+                id_param: self.uid,
+                metadata_param: include_metadata
+            })
             res = res["exportBatchDataRows"]
             if res["status"] == "COMPLETE":
                 download_url = res["downloadUrl"]
                 response = requests.get(download_url)
                 response.raise_for_status()
                 reader = ndjson.reader(StringIO(response.text))
-                # TODO: Update result to parse metadataFields when resolver returns
-                return (Entity.DataRow(self.client, {
-                    **result, 'metadataFields': [],
-                    'customMetadata': []
-                }) for result in reader)
+                return (Entity.DataRow(
+                    self.client, {
+                        **result, 'customMetadata': result['metadata'],
```

Comment:
I think this should be `result['customMetadata']`, but can you double check by printing the result?

Reply:
`DataRow {'created_at': datetime.datetime(2022, 6, 1, 21, 13, 28, tzinfo=datetime.timezone.utc), 'external_id': '5668a62497e6edcd', 'media_attributes': {}, 'metadata': [DataRowMetadataField(schema_id='cl3w2pwsr000nj6ouhqebbf19', value=datetime.datetime(2021, 10, 13, 21, 1, 44, 34911, tzinfo=datetime.timezone.utc))], 'metadata_fields': [{'schemaId': 'cl3w2pwsr000nj6ouhqebbf19', 'name': 'capture_date', 'kind': 'CustomMetadataDateTime', 'value': '2021-10-13T21:01:44.034911Z'}], 'row_data': 'https://storage.googleapis.com/labelbox-datasets/open-images/data/5668a62497e6edcd.jpg', 'uid': 'cl3w355rxh5doj6ou1ftpffq7', 'updated_at': datetime.datetime(2022, 6, 1, 21, 13, 28, tzinfo=datetime.timezone.utc)}> `

This is an example result. i think we said that customMetadata -> `metadata` which should be the flattened view while `metadata_fields` is the simplified version